### ISSUE TITLE
Suggestion - replace custom mathjax with mathjax-electrong project

### ISSUE THREAD

**Issue by mangecoeur** [atom-community/markdown-preview-plus#227]
The guys over at nteract project have put together an npm-installable mathjax-election bundle

https://github.com/nteract/mathjax-electron/

Could be nice to use that instead of home-rolled mathjax. Plan to do the same for preview-inline at some point. One advantage is getting an up-to-date version of mathjax with less faffing.

**Comment by mangecoeur** [comment]
p.s. @Galadirith I saw a while back that you didn't have much time to work on this. I don't have much time either, but would be happy to help however I can if you want to add me to project I could filter a few issues/PRs - I use this project pretty much daily so keen to see it live on ;)

**Comment by lierdakil** [comment]
Problem is, it's not configurable, so no user-defined macros, no possibility for ASCIIMath, etc. So... yeah... the choices are, pretty much, either fork *that*, stick with home-rolled, or switch to mathjax-node.
